src/components/FormSections/PoliticalParty.jsx

```diff
@@ -21,7 +20,7 @@ function PoliticalParty(props){
         <>
         <h2>{headings.step_label_4}</h2>
 
-        {(partyStateInstructions || partyGeneralInstructions) && (
```

Comment:
Revert this - we want the general instructions to appear if there are no state specific instructions to override